src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/EventSymbolReferenceFinder.cs

```diff
@@ -24,6 +24,8 @@ protected override bool CanFind(IEventSymbol symbol)
             var baseSymbols = await base.DetermineCascadedSymbolsAsync(
                 symbol, solution, projects, options, cancellationToken).ConfigureAwait(false);
 
+            // TODO: This probably indicates a problem because event backing fields have not been in GetMembers() for a
+            // long time. Compare to the changes that were necessary in PropertySymbolReferenceFinder.
```

Comment:
I think there is an existing 'find all references' test hole here that would require IEventSymbol.AssociatedField to be exposed.